src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -0,0 +1,441 @@
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>1. Empty Cell</td>
+      <td>Cells with no values should be left blank. Do not use a hyphen, en, or em dash.</td>
+    </tr>
+    <tr>
+      <td>2. Hover</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>3. Focused</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>4. Selected</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>5. Focused and Selected</td>
+      <td>
+        The last row checked is the focused row, unless the user clicks on a different row or uses
+        the keyboard to change focus.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Loading
+
+Skeleton screens are used to indicate that a screen is loading. They are perceived as being shorter in duration when
+compared against a blank screen or a spinner.
+
+- Implement the [Skeleton](/components/skeleton) component in each cell up to three rows
+- For a table with three or more rows, show the first three rows as skeletons with the header cells being shown at all
+  times to provide instant context of incoming tabular data
+- The width of the Skeleton should match the width of the intended cell content, while the Skeleton’s height will be 60%
+  of the font size
+
+import TableLoading from '../examples/tables/TableLoading.tsx';
+import TableLoadingCode from '!!raw-loader!../examples/tables/TableLoading.tsx';
+
+<CodeExample code={TableLoadingCode}>
+  <TableLoading />
+</CodeExample>
+
+## Localization & internationalization
+
+Our tables have been designed to accommodate localization and internationalization standards so they do not break the
+experience.
+
+<figure>
+  <div style="border:1px solid #E5E5E5;">
+    <img
+      src="figma:patterns-tables-basic-formatting-localization-bulgarian.png"
+      alt="Bulgarian localization example"
+    />
+  </div>
+  <figcaption align="center" style="text-align:center;color:#68737D;margin-top:8px;">
+    Bulgarian example
+  </figcaption>
+</figure>
+
+<br />
+
+<figure>
+  <div style="border:1px solid #E5E5E5;">
+    <img
+      src="figma:patterns-tables-basic-formatting-localization-arabic.png"
+      alt="Arabic localization example"
+    />
+  </div>
+  <figcaption align="center" style="text-align:center;color:#68737D;margin-top:8px;">
+    Arabic RTL example
+  </figcaption>
+</figure>
+
+## Accessibility
+
+### General considerations
+
+All Garden [Tables](/components/table) are built with accessible markup.
+
+### Screen Reader order
+
+People using screen readers can have the row and column cells read aloud as they navigate through the table. Screen
+readers speak one cell at a time and reference the associated header cells, so the reader doesn’t lose context.
+
+### Additional resources
+
+- [https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced)
+
+---
+
+<h2>Feedback</h2>
+
+Help improve this page. [Give us your feedback](https://forms.gle/VFD799SvNfs2yiFC7).
```

Comment:
@hzhu please connect with me offline re: the form before this PR is merged. I need to do a permissions audit.

Reply:
Trying to understand how much effort this will be exactly @hzhu / @jzempel. 

If it is more effort than initially anticipated, could we break it out into a separate fast-follow ticket? Any concerns with that @james-sann / @nyckie ?